### DISCUSSION TITLE
first time user needs help; Mac OS

### DISCUSSION THREAD

**Discussion by David72645** [Netatalk/netatalk#1594] [Q&A] [answered]
I'm trying to install Netatalk 4 on Sonoma; OS 14.7. I'm trying to follow [Instructions](https://netatalk.io/stable/htmldocs/compile#build-macos) but at step two I get this error "Neither source directory 'build' nor build directory None contain a build file meson.build." I know the instructions are for people more advanced them me, but is there a step by step that is more detailed so I don't make basic mistakes (something that spell everything out)? Thanks.

**Comment by rdmark** [comment]
@David72645 At step 2, did you copy paste and execute all three lines in one go?

```
meson setup build \
  -Dwith-appletalk=true \
  -Dwith-tests=true
```

The backslash means a continuation of the prior line on a terminal.

**Comment by David72645** [comment]
Thank you. Yes I copied all three lines and ran as one go.  I did not know backspash was continuation. Good to know. I'm wondering if I'm supposed to do before I start the install like creating a folder?  I did try to use Chat GPT to help me install and spent about an hour going around in circles. I have some experience with Brew bu tot Meson.

**Comment by rdmark** [comment, marked as answer]
Got it, so you're missing the step where you download a release tarball, or clone the git repository.

Please read through and follow the steps in this manual chapter first: https://netatalk.io/stable/htmldocs/installation

Once you finished this, go back to the macOS compilation chapter.

**Comment by David72645** [comment]
I did get it installed and running, but now I can't make it do what I want. Should I start a new thread for that or give you details. Thanks for your help and doing this project.

**Reply by rdmark** [reply]
Please start a new thread!
